android-project/app/src/main/java/org/libsdl/app/SDLActivity.java

```diff
@@ -53,6 +53,8 @@
 import android.widget.RelativeLayout;
 import android.widget.TextView;
 import android.widget.Toast;
+import android.text.Editable;
+import android.text.Selection;
```

Comment:
Can you add these in alphabetic order?
They should be added before and after `import android.text.InputType;`.